ArcGISRuntimeSDKQt_CppSamples/Maps/GenerateOfflineMap_Overrides/OverridesWindow.qml

```diff
@@ -0,0 +1,257 @@
+// Copyright 2017 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import QtQuick 2.6
+import QtQuick.Controls 2.2
+
+Rectangle {
+    id: overridesPanel
+    visible: overridesReady
+    signal basemapLODSelected(real min, real max)
+    signal basemapBufferChanged(real buffer)
+    signal removeSystemValvesChanged()
+    signal removeServiceConnectionChanged()
+    signal hydrantWhereClauseChanged(string whereClause)
+    signal clipWaterPipesAOIChanged(bool clip)
+    signal overridesAccepted()
+
+    color: "#D6D6D6"
+
+    Text {
+        id: title
+        anchors {
+            top: parent.top
+            horizontalCenter: parent.horizontalCenter
+            margins: 16 * scaleFactor
+        }
+        text: "Layer specific overrides"
+        font{
+            bold: true
+            underline: true
+            pixelSize: 18 * scaleFactor
+        }
+        color: "#474747"
+    }
+
+    ScrollView {
+        id: scrollView
+        anchors {
+            top: title.bottom
+            left: overridesPanel.left
+            right: overridesPanel.right
+            bottom: takeOfflineButton.top
+            margins: 16 * scaleFactor
+        }
+        clip: true
+        ScrollBar.vertical.interactive: true
+        ScrollBar.vertical.policy: ScrollBar.AsNeeded
+
+        Rectangle {
+            implicitHeight: childrenRect.height
+            implicitWidth: overridesPanel.width
+            color: "transparent"
+
+            Text {
+                id: lodTitle
+                text: "Basemap Levels of Detail:"
+                anchors{
```

Comment:
we usually have a space between the property and the curly brace - `anchors {`. This is throughout this file